plugins/gradle/java/src/execution/build/GradleBaseApplicationEnvironmentProvider.kt

```diff
@@ -238,6 +244,16 @@ abstract class GradleBaseApplicationEnvironmentProvider<T : JavaRunConfiguration
       this.sourceSetName = sourceSetName
       return this
     }
+
+    fun withUseManifestJar(useManifestJar: Boolean?): GradleInitScriptParametersBuilder {
+      this.useManifestJar = useManifestJar
+      return this
+    }
+
+    fun withUseArgsFile(useArgsFile: Boolean?): GradleInitScriptParametersBuilder {
+      this.useArgsFile = useArgsFile
+      return this
+    }
```

Comment:
Could you, please, instead extract the type of shortenning directly from `org.jetbrains.plugins.gradle.execution.build.GradleInitScriptParameters#configuraton#shortenCommandLine`. You can refer to the `org.jetbrains.plugins.gradle.execution.build.GradleApplicationEnvironmentProvider`.

Reply:
Done